cordova-lib/src/cordova/metadata/windows_parser.js

```diff
@@ -65,135 +53,13 @@ require('util').inherits(windows_parser, Parser);
-            throw new Error('update_from_config expected a valid package.appxmanifest' +
-                            ' with a <VisualElements> node');
-        }
-    }
-    else {
-        throw new Error('update_from_config expected a valid package.appxmanifest' +
-                        ' with a <Application> node');
-    }
-
-    // Update properties
-    var properties = manifest.find('.//Properties');
-    if (properties && properties.find) {
-        var displayNameElement = properties.find('.//DisplayName');
-        if (displayNameElement && displayNameElement.text != name) {
-            displayNameElement.text = name;
-        }
-
-        var publisherNameElement = properties.find('.//PublisherDisplayName');
-        if (publisherNameElement && publisherNameElement.text != author) {
-            publisherNameElement.text = author;
-        }
-    }
-
-    // sort Capability elements as per CB-5350 Windows8 build fails due to invalid 'Capabilities' definition
-    // to sort elements we remove them and then add again in the appropriate order
-    var capabilitiesRoot = manifest.find('.//Capabilities'),
-        capabilities = capabilitiesRoot._children || [];
-
-    capabilities.forEach(function(elem){
-        capabilitiesRoot.remove(elem);
-    });
-    capabilities.sort(function(a, b) {
-        return (a.tag > b.tag)? 1: -1;
-    });
-    capabilities.forEach(function(elem){
-        capabilitiesRoot.append(elem);
-    });
-
-    //Write out manifest
-    fs.writeFileSync(this.manifestPath, manifest.write({indent: 4}), 'utf-8');
-
-    // Update icons
-    var icons = config.getIcons('windows8');
-    var platformRoot = this.projDir;
-    var appRoot = util.isCordova(platformRoot);
-
-    // Icons, that should be added to platform
-    var platformIcons = [
-        {dest: 'images/logo.png', width: 150, height: 150},
-        {dest: 'images/smalllogo.png', width: 30, height: 30},
-        {dest: 'images/storelogo.png', width: 50, height: 50},
-    ];
-
-    platformIcons.forEach(function (item) {
-        var icon = icons.getBySize(item.width, item.height) || icons.getDefault();
-        if (icon){
-            var src = path.join(appRoot, icon.src),
-                dest = path.join(platformRoot, item.dest);
-            events.emit('verbose', 'Copying icon from ' + src + ' to ' + dest);
-            shell.cp('-f', src, dest);
-        }
-    });
-
-    // Update splashscreen
-    // Image size for Windows 8 should be 620 x 300 px
-    // See http://msdn.microsoft.com/en-us/library/windows/apps/hh465338.aspx for reference
-    var splash = config.getSplashScreens('windows8').getBySize(620, 300);
-    if (splash){
-        var src = path.join(appRoot, splash.src),
-            dest = path.join(platformRoot, 'images/splashscreen.png');
-        events.emit('verbose', 'Copying icon from ' + src + ' to ' + dest);
-        shell.cp('-f', src, dest);
-    }
+    // If there is platform-defined prepare script, require and exec it
```

Comment:
Please update the comment accordingly, as now the platform _must_ contain `prepare` script